src/main/java/com/dezorganizacja/bridge/view/graphical/Controller.java

```diff
@@ -0,0 +1,11 @@
+package com.dezorganizacja.bridge.view.graphical;
+
+import com.dezorganizacja.bridge.presenter.MainPresenter;
+
+public class Controller {
+    protected MainPresenter presenter;
```

Comment:
ok, teraz pytanie, skąd będzie można się dowiedzieć, który presenter jest aktualnie aktywny / którego używać? obstawiam, że to powinno być wysyłane razem ze state z MainPresentera?